packages/gatsby-plugin-manifest/src/__tests__/gatsby-node.js

```diff
@@ -159,6 +156,7 @@ describe(`Test plugin manifest options`, () => {
       plugins: [],
       theme_color_in_head: false,
       cache_busting_mode: `name`,
+      icon_options: {},
```

Comment:
This doesn't _need_ to be passed right? i.e. it's tolerant if this isn't passed.

Reply:
The whole point of this test is to make sure these extra properties don't show up in what's written to the manifest. I needs to be there so it can be deleted.

Reply:
This test should probably be re-written to make that more clear then, e.g. something like

```
const manifestOptions = {
  theme_color_in_head: false,
  cache_busting_mode: `name`
}

const iconOptions = {
  icon_options: {},
  something_else: true
}

// call the method
// assert no extra options were passed
expect(whatever).toEqual(manifestOptions)
```

otherwise it's sorta testing a side effect. This test could fail in the future, and it's not clear _why_ it's failing because it's not clear what it's testing.